astropy/io/fits/fitstime.py

```diff
@@ -65,6 +65,11 @@ def is_time_column_keyword(keyword):
                     'TREFPOS': ('TOPOCENTER', 'Time reference position')}
 
 
+# FITS time unit to astropy format conversion
+UNIT_TO_FORMAT = {'s': 'sec',
+                  'd': 'jd'}
```

Comment:
I don't understand why this is needed, since the `TimeDelta` that is initiated is never returned, but always added to a reference time. I would remove it altogether.